test/index.js

```diff
@@ -55,3 +55,21 @@ test('params passed to get() work correctly', (t) => {
     t.is(res[0].slug, 'hello-world')
   })
 })
+
+test('uses (http) url instead of name', (t) => {
```

Comment:
I think it could be redundant, but it also verifies that `http` is available if the protocol logic were to ever change